web/src/test/java/org/cbioportal/web/util/DataBinnerTest.java

```diff
@@ -48,608 +48,10 @@
 
     @Before
     public void setup() {
-        mockData = initMockData();
+        mockData = DataBinnerMocker.mockData();
         MockitoAnnotations.initMocks(this);
     }
 
-    public Map<String, String[]> initMockData() {
```

Comment:
Cannot add anymore data into this method due to a compiler limit (https://stackoverflow.com/a/2407930). Moved mock data into `DataBinnerMocker.java` instead.